docs/Versions.md

```diff
@@ -0,0 +1,21 @@
+---
+id: versions
+title: WebdriverIO Versions
+---
+
+The project team releases new major versions roughly on a yearly cadence. LTS release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 12 months until a new major release is made.
+
+## Current version (Stable)
+
+| | | | |
+|----|--------------------|--------------------------------------------------------------------------------------|-----------------|
+| v6 | [Documentation](/) | [Release Notes](https://github.com/webdriverio/webdriverio/blob/master/CHANGELOG.md) | __Stable__ |
+
+## Past Versions
+
+Here you can find documentation for previous versions of WebdriverIO.
+
+| | | | |
+|----|-------------------------------------------|----------------------------------------------------------------------------------------------|-----|
+| v5 | [Documentation](https://v5.webdriver.io/) | [Release Notes](https://github.com/webdriverio/webdriverio/blob/v5/CHANGELOG.md#changelog) | __LTS__ (until January 2021) |
```

Comment:
We don't have a v5 link yet, should we fix this first or wait till v6?

Reply:
yeah , this was a configuration issue which I just fixed.